config/locales/en.yml

```diff
@@ -1811,6 +1811,11 @@ en:
       recent_notification_description: 'Recent changes notifications appear whenever there is a change on a task you are assigned to.'
       system_message: 'System message'
       system_message_description: 'System message notifications are specifically sent by site maintainers to notify all users about a system update.'
+    deliver:
+      download_link: "Download link:"
+      email_subject: "Your SciNote export is ready!"
+      email_body: "<p>The export of SciNote project(s) that you requested is ready!</p><p>You can find the link to download the file below or in your SciNote notifications. Please keep in mind that the link will expire in 7 days for security reasons.</p>"
```

Comment:
This text is specific to "export all" export type, and since it is used for all kinds of exports, it should either be generalized in a way that could be used with all of them, or you should make different locales for each export type.

Reply:
I'm just following what I've been told. There's one email template to all different types of exports and that is it. I asked this on the meeting.